src/plugin/ipc/file/ptyconnection.cpp

```diff
@@ -477,6 +488,10 @@ PtyConnection::postRestart()
     }
   }
 
+  // Apparently, the PtyConnection object has a private vector of _fds
```

Comment:
FYI: it's not just PtyConnection, but Connection class itself that contains the _fds vector.